src/Symfony/Component/Debug/Tests/FatalErrorHandler/ClassNotFoundFatalErrorHandlerTest.php

```diff
@@ -115,13 +134,19 @@ public function provideClassNotFoundData()
 
     public function provideLegacyClassNotFoundData()
     {
+        $this->iniSet('error_reporting', -1 & ~E_USER_DEPRECATED);
+
         $prefixes = array('Symfony\Component\Debug\Exception\\' => realpath(__DIR__.'/../../Exception'));
 
         $symfonyAutoloader = new SymfonyClassLoader();
         $symfonyAutoloader->addPrefixes($prefixes);
 
-        $symfonyUniversalClassLoader = new SymfonyUniversalClassLoader();
-        $symfonyUniversalClassLoader->registerPrefixes($prefixes);
+        if (class_exists('Symfony\Component\ClassLoader\UniversalClassLoader')) {
+            $symfonyUniversalClassLoader = new SymfonyUniversalClassLoader();
+            $symfonyUniversalClassLoader->registerPrefixes($prefixes);
+        } else {
+            $symfonyUniversalClassLoader = $symfonyAutoloader;
```

Comment:
shouldn't you ignore this test in this case instead of transforming it into a duplicate test by reusing the non-deprecated loader ?